Tasks/SqlAzureDacpacDeployment/Utility.ps1

```diff
@@ -27,21 +27,27 @@ function Get-AgentIPRange
 
     $ErrorActionPreference = 'Continue'
 
-    ( Invoke-Expression "& '$sqlCmd' --% $sqlCmdArgs" -ErrorVariable errors -OutVariable output 2>&1 ) | Out-Null   
+    $output = ( Invoke-Expression "& '$sqlCmd' --% $sqlCmdArgs" -ErrorVariable errors 2>&1 ) | Out-String
 
     $ErrorActionPreference = 'Stop'
 
     if($errors.Count -gt 0)
```

Comment:
No this is for not stopping task on error. We wanted to continue the execution of the script